backend/game_server/lib/cell.ex

```diff
@@ -0,0 +1,32 @@
+defmodule BernacleServer.Cell do
+	alias BernacleServer.{Entity,Cell}
+	alias BernacleServer.Helpers.{Physic,Vector}
+
+
+	def born(position, velocity) do
+		Entity.new
+			|> Entity.set_attribute(:position, position)
+			|> Entity.set_attribute(:velocity, velocity)
+			|> Entity.set_attribute(:mass, 5)
+			|> Entity.give_ability(:move, Cell)
+			|> Entity.give_ability(:eat, Cell)
+
+	end
+
+	def born(position), do: born(position, %Vector{})
+
+	def born(), do: born(%Vector{}, %Vector{x: 1, y: 1})
```

Comment:
it's will be confusing when if you call born with one argument it will not have velocity, but with zero argument - have `x: 1, y: 1` velocity.